resources/services/elasticbeanstalk/application_versions.go

```diff
@@ -0,0 +1,139 @@
+func ApplicationVersions() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_elasticbeanstalk_application_versions",
+		Description:  "Describes the properties of an application version.",
+		Resolver:     fetchElasticbeanstalkApplicationVersions,
+		Multiplex:    client.ServiceAccountRegionMultiplexer("elasticbeanstalk"),
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"arn"}},
+		Columns: []schema.Column{
+			{
+				Name:        "account_id",
+				Description: "The AWS Account ID of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSRegion,
+			},
+			{
+				Name:        "application_name",
+				Description: "The name of the application to which the application version belongs.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the application version.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("ApplicationVersionArn"),
+			},
+			{
+				Name:          "build_arn",
+				Description:   "Reference to the artifact from the AWS CodeBuild build.",
+				Type:          schema.TypeString,
+				IgnoreInTests: true,
+			},
+			{
+				Name:        "date_created",
+				Description: "The creation date of the application version.",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "date_updated",
+				Description: "The last modified date of the application version.",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "description",
+				Description: "The description of the application version.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:          "source_build_information_source_location",
+				Description:   "The location of the source code, as a formatted string, depending on the value of SourceRepository  * For CodeCommit, the format is the repository name and commit ID, separated by a forward slash",
+				Type:          schema.TypeString,
+				Resolver:      schema.PathResolver("SourceBuildInformation.SourceLocation"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:        "source_build_information_source_repository",
+				Description: "Location where the repository is stored.  * CodeCommit  * S3  This member is required.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("SourceBuildInformation.SourceRepository"),
+			},
+			{
+				Name:        "source_build_information_source_type",
+				Description: "The type of repository.  * Git  * Zip  This member is required.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("SourceBuildInformation.SourceType"),
```

Comment:
nit: would remove the prefix here

Reply:
Good point! Done in https://github.com/cloudquery/cq-provider-aws/pull/751/commits/b2d0102889ee8376bae07fa241cda78e406fc051